### DISCUSSION TITLE
set_e820_map function may be useless

### DISCUSSION THREAD

**Discussion by yoshinoyatoko** [memtest86plus/memtest86plus#422] [General]
Hello,
For uefi app execute environment, boot_params struct's efi_info is enough for labeling current memory range,
maybe set_e820_map function is useless?

**Comment by martinwhitaker** [comment]
If `set_820_map` was removed `pmem.c` would have to be modified to parse both EFI and E820 memory maps. So this would save no code.

**Reply by yoshinoyatoko** [reply]
Hi, Martin;
Got it!

pmem.c will leverage e820 list.
